openstack/cronus/templates/rhea/_config.yaml.tpl

```diff
@@ -19,6 +19,23 @@ rhea:
     endpointType: {{ .Values.config.endpointType }}
 {{- end }}
   auth:
+    keystone:
+{{- if .Values.config.keystone }}
+{{- range $key, $value := .Values.config.keystone }}
+    {{- if $value }}
+      {{ $key }}: {{ $value }}
+    {{- end }}
+{{- end }}
+{{- if .Values.global.cronus_service_password }}
+      password: {{ .Values.global.cronus_service_password }}
+{{- end }}
+{{ else }}
+      authUrl: {{ .Values.config.authUrl }}
+      applicationCredentialID: {{ .Values.config.applicationCredentialID }}
+      applicationCredentialSecret: {{ .Values.config.applicationCredentialSecret }}
+      region: {{ .Values.config.region }}
```

Comment:
FYI, Region should already be set in the globals so as to not accidentally copy paste. I.E. `{{.Values.global.region}}`

Reply:
Addressing in an upcoming ticket